src/components/DefaultLayout/DefaultLayout.js

```diff
@@ -16,13 +22,21 @@ const DefaultLayout = ({
   <Route
     {...rest}
     render={(matchProps) => (
-      <>
-        {!hideHeader && <Header />}
-        <Component {...matchProps} />
-        {!hideBrands && <Brands />}
-        {!hideShippingInfo && <ShippingInfo />}
-        {!hideFooter && <Footer />}
-      </>
+      <TransitionGroup>
+        <CSSTransition
+          key={location.key}
+          timeout={300}
+          classNames="fade"
+        >
+          <>
+            {!hideHeader && <Header />}
```

Comment:
Just FYI if mostly these 'hide_' constants are false maybe better to rename them to 'show_' and check for true statement, not false one

Reply:
I am in progress of refactoring this component on another branch) Let it be here in this way)
But I promise I will correct it!:)

Reply:
Just keep it in mind for the next time:)